### ISSUE TITLE
Trivial test fails.

### ISSUE THREAD

**Issue by vi** [BartMassey/mixed-strategies-rs#6]
```
$ printf -- '0\n   0\n' | mixed-strategies  
thread 'main' panicked at 'assertion failed: v > 0.0', /home/rust/.cargo/git/checkouts/mixed-strategies-rs-67b1db906ef82bc0/a2e9fc3/src/lib.rs:366:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

There's no choice at all for both players. I expect output like this:

```
value 0.000
max 
min  
```

**Comment by BartMassey** [comment]
Interesting bug, thanks!

I think the matrix you gave actually has a maximizer choice but no minimizer choice: one column with two rows, they are just both the same.

The crash happens because the value calculation doesn't want to divide by zero. I'll have to thank about how to fix this — degenerate cases are confusing.

Again, thanks much for catching this.

**Comment by vi** [comment]
Actual example was supposed to be 1x1 matrix:

```
$ printf '0\n' | mixed-strategies 
thread 'main' panicked
```

Basically any zero matrix fails:

```
$ printf '0 0 0 0\n0 0 0 0\n0 0 0 0\n0 0 0 0\n' | mixed-strategies 
thread 'main' panicked at 
```

**Comment by BartMassey** [comment]
Any matrix with all-equal values, yes. This is a sort of degenerate case in that you can play however you like, since all outcomes are equal: the solver doesn't handle it well. For now I've changed the interface to return `None` as the solution in this case. Maybe not the best fix, but works around the immediate problem.

Again thanks much. I'll keep thinking on whether there's a better plan.

"fixed" in commit bcdbbc7
